apidoc/Titanium/UI/Android/Android.yml

```diff
@@ -30,6 +30,142 @@ methods:
         format.
         
 properties:
+  - name: GRAVITY_AXIS_CLIP
+    summary: Raw bit controlling whether the right/bottom edge is clipped to its container, based on the gravity direction being applied.
+    type: Number
+    permission: read-only
+    since: "6.0.0"
```

Comment:
The "since" is for the Ti-version, not the Android version. Can you correct all constants based on your current knowledge? You could make a "description" param that says something like "Only available on Android 6 and later". If some are available earlier / later, go ahead adjust it.